rascaline-c-api/rascaline.h

```diff
@@ -52,6 +52,8 @@ typedef struct rascal_calculator_t rascal_calculator_t;
  */
 typedef struct rascal_descriptor_t rascal_descriptor_t;
 
+typedef void (*RascalLoggingCallback)(int32_t level, const char *message);
```

Comment:
I would rather use C naming convention for this C type, i.e. 
```suggestion
typedef void (*rascal_logging_callback_t)(int32_t level, const char *message);
```

Reply:
I think if you rename it here, you have to rename it also in the rust file, and then you get a warning

Reply:
Yes, but that's a warning we can ignore 😄. You can use something like 

```rust
#[allow(non_camel_case_types)]
pub type rascal_logging_callback_t = Option<unsafe extern fn(level: i32, message: *const std::os::raw::c_char)>;
```